test/integration/component/test_acl_listvolume.py

```diff
@@ -357,9 +357,9 @@ def tearDownClass(cls):
         cls.apiclient = super(TestVolumeList, cls).getClsTestClient().getApiClient()
         cls.apiclient.connection.apiKey = cls.default_apikey
         cls.apiclient.connection.securityKey = cls.default_secretkey
+        cleanup_resources(cls.apiclient, cls.cleanup)
```

Comment:
Is this to fix this problem?  This is from #1297, which was ran before I got this fix merged.

```
======================================================================
ERROR: test suite for <class 'integration.component.test_acl_listvm.TestVMList'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/.local/lib/python2.7/site-packages/nose/suite.py", line 228, in run
    self.tearDown()
  File "/home/travis/.local/lib/python2.7/site-packages/nose/suite.py", line 351, in tearDown
    self.teardownContext(ancestor)
  File "/home/travis/.local/lib/python2.7/site-packages/nose/suite.py", line 367, in teardownContext
    try_run(context, names)
  File "/home/travis/.local/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/home/travis/build/apache/cloudstack/test/integration/component/test_acl_listvm.py", line 346, in tearDownClass
    cls.domain_1.delete(cls.apiclient,cleanup="true")
  File "/home/travis/.local/lib/python2.7/site-packages/marvin/lib/base.py", line 78, in delete
    apiclient.deleteDomain(cmd)
  File "/home/travis/.local/lib/python2.7/site-packages/marvin/cloudstackAPI/cloudstackAPIClient.py", line 703, in deleteDomain
    response = self.connection.marvinRequest(command, response_type=response, method=method)
  File "/home/travis/.local/lib/python2.7/site-packages/marvin/cloudstackConnection.py", line 379, in marvinRequest
    raise e
Exception: Job failed: {jobprocstatus : 0, created : u'2016-05-13T14:25:30+0000', cmd : u'org.apache.cloudstack.api.command.admin.domain.DeleteDomainCmd', userid : u'7e18b4b2-1915-11e6-abc2-42010a800044', jobstatus : 2, jobid : u'a3e58407-3f1a-4508-9671-733029d283c0', jobresultcode : 530, jobresulttype : u'object', jobresult : {errorcode : 530, errortext : u'Failed to clean up domain resources and sub domains, delete failed on domain D1-HVOVF7 (id: 2).'}, accountid : u'7e188b32-1915-11e6-abc2-42010a800044'}
```